internal/provider/onepassword_item_data_source.go

```diff
@@ -298,6 +332,19 @@ func (d *OnePasswordItemDataSource) Read(ctx context.Context, req datasource.Rea
 		}
 	}
 
+	for _, f := range item.Files {
+		content, err := d.client.GetFileContent(ctx, f, item.ID, item.Vault.ID)
```

Comment:
`[nitpick]`

What might be useful here is to call the function if the file's content doesn't exist yet, which is the case for Connect, but not for the CLI.

Reply:
fixed